f5xc/ce/gcp/nodes/outputs.tf

```diff
@@ -7,8 +7,8 @@ output "ce" {
       labels      = google_compute_instance.instance.labels
       sli_ip      = var.f5xc_ce_gateway_type == var.f5xc_ce_gateway_type_ingress_egress ? google_compute_instance.instance.network_interface[1].network_ip : null
       slo_ip      = google_compute_instance.instance.network_interface[0].network_ip
-      public_ip   = google_compute_instance.instance.network_interface[0].access_config[0].nat_ip
+      public_ip   = var.use_public_ip ? google_compute_instance.instance.network_interface[0].access_config[0].nat_ip : ""
```

Comment:
Not sure if this should be an "empty string" or `null` ... I couldn't tell where it was being used.